weblate/trans/tests/test_hooks.py

```diff
@@ -106,40 +107,73 @@
 
 GITLAB_PAYLOAD = """
```

Comment:
```suggestion
GITLAB_PAYLOAD = r"""
```

The test fails because of wrong JSON (\n is posted as newline), this fixes it.